test/CreateTest.php

```diff
@@ -34,33 +42,34 @@ public function test_01_create_test_ping_and_close() {
 			$c->close();
 		}
 
+		/**
+		 * @doesNotPerformAssertions
+		 */
 		public function test_01_01_double_disconnect() {
 			$a = new Tarantool('localhost', self::$port);
 			$a->disconnect();
 			$a->disconnect();
 		}
 
-		/**
-		 * @expectedException TarantoolException
-		 * @expectedExceptionMessageRegExp /Name or service not known|nodename nor servname provided/
-		 */
 		public function test_02_create_error_host() {
+			$this->expectException(TarantoolException::class);
```

Comment:
Nit: It is better to place these expectations as close as possible to the method that should throw the exception, in the way it's written now, the constructor can throw the same exception as `connect()` and the test will pass (that's very unlikely in this particular case, but may become an issue later if someone blindly copies this code). So I would write:

```php
$tarantool = new Tarantool('very_bad_host');

$this->expectException(TarantoolException::class);
$this->expectExceptionMessageMatches(...);
$tarantool->connect();
```

Reply:
Agreed, will fix.